src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -71,12 +73,12 @@ public PersonCard(ReadOnlyPerson person, int displayedIndex) {
     private void bindListeners(ReadOnlyPerson person) {
         name.textProperty().bind(Bindings.convert(person.nameProperty()));
         phone.textProperty().bind(Bindings.convert(person.phoneProperty()));
-        homeNumber.textProperty().bind(Bindings.convert(person.homeNumberProperty()));
-        address.textProperty().bind(Bindings.convert(person.addressProperty()));
         email.textProperty().bind(Bindings.convert(person.emailProperty()));
-        schEmail.textProperty().bind(Bindings.convert(person.schEmailProperty()));
-        website.textProperty().bind(Bindings.convert(person.websiteProperty()));
-        birthday.textProperty().bind(Bindings.convert(person.birthdayProperty()));
+        //homeNumber.textProperty().bind(Bindings.convert(person.homeNumberProperty()));
+        //address.textProperty().bind(Bindings.convert(person.addressProperty()));
+        //schEmail.textProperty().bind(Bindings.convert(person.schEmailProperty()));
+        //website.textProperty().bind(Bindings.convert(person.websiteProperty()));
+        //birthday.textProperty().bind(Bindings.convert(person.birthdayProperty()));
```

Comment:
Don't leave commented out code. Just go ahead and delete unused code. :-)

Reply:
Hi Prof,
Yeap we deleted this in v1.4